.env.example

```diff
@@ -1,11 +1,17 @@
 DISCORD_TOKEN=
 CHANNEL_ID=
 
-FAUCET_MNEMONIC=
+FAUCET_MNEMONIC=likecoin-public-testnet-5
```

Comment:
Why `mnemonic` is `likecoin-public-testnet-5`?
If it's just an example / placeholder, please give an example with the expected format (e.g. for mnemonic it should be 12 / 24 words?)

Reply:
Umm.. typo, fixed